File: Bimestre1/EvaBimestral/EvaBimestralPython/cajaFuerte.py
print("Caja fuerte")
correct = int(2032)
n = int(4)
i = int(1)
clave = int(0)
# proceso
clave = int(input("Ingrese la clave: "))
if correct == clave:
    print("La caja fuerte se ha abierto satisfactoriamente")
if correct != clave:
    print("Lo siento, esa no es la combinación")
while (i < n) and (correct != clave):
    clave = int(input("Ingrese la clave: "))
    if correct == clave:
        print("La caja fuerte se ha abierto satisfactoriamente")
    if correct != clave:

        print("Lo siento, esa no es la combinación")
    i=i+1
    if i >3:
        print("Demasiados intentos")
        print("La caja fuerte ha sido bloqueada")

File: Bimestre1/EvaBimestral/EvaBimestral1/src/evabimestral1/CajaFuerte.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package evabimestral1;
import java.util.Scanner;
/**
 *
 * @author kevin
 */
public class CajaFuerte {
    public static void main(String[] args) {
        Scanner leer = new Scanner(System.in);
        int correct, n, clave,i;
        correct=2032; n=4; i=1; clave=0;
        
       //proceso
       
        do{
            System.out.print("Ingrese la clave: ");
            clave= leer.nextInt();
            if(correct==clave){
                System.out.println("La caja fuerte se ha abierto satisfactoriamente");
            }
            if(correct!=clave){
                System.out.println("Lo siento, esa no es la combinación");
            }
            i=i+1; 
            if((i>4)){
                System.out.println("Demasiados intentos");
                System.out.println("La caja fuerte ha sido bloqueada");
            }
             
               
           } while ((i<=n)&&(correct != clave));
    }
}
